src/sentry/static/sentry/app/views/eventsV2/table/cellAction.tsx

```diff
@@ -113,16 +133,64 @@ export default class CellAction extends React.Component<Props, State> {
         }
         query[negation].push(`${value}`);
         break;
+      case Actions.SHOW_GREATER_THAN: {
+        // Remove query token if it already exists
+        delete query[`${column.name}`];
+        query[column.name] = [`>${value}`];
+        const field = {field: column.name, width: column.width};
+
+        // sort descending order
+        nextView = nextView.withSorts([fieldToSort(field, tableMeta, 'desc')!]);
+
+        break;
+      }
+      case Actions.SHOW_LESS_THAN: {
+        // Remove query token if it already exists
+        delete query[`${column.name}`];
+        query[column.name] = [`<${value}`];
+        const field = {field: column.name, width: column.width};
+
+        // sort ascending order
+        nextView = nextView.withSorts([fieldToSort(field, tableMeta, 'asc')!]);
+
+        break;
+      }
+      case Actions.TRANSACTION: {
+        const maybeProject = projects.find(project => {
+          return project.slug === dataRow.project;
+        });
+
+        const projectID = maybeProject
+          ? [maybeProject.id]
+          : eventView.project.map(id => String(id));
+
+        const next = transactionSummaryRouteWithQuery({
+          orgSlug: organization.slug,
+          transaction: String(value),
+          projectID,
+          query: {},
+        });
+
+        browserHistory.push(next);
+        return;
+      }
+      case Actions.RELEASE: {
+        const maybeProject = projects.find(project => {
+          return project.slug === dataRow.project;
+        });
+
+        browserHistory.push({
+          pathname: `/organizations/${organization.slug}/releases/${encodeURIComponent(
+            value
+          )}/`,
+          query: {project: maybeProject ? maybeProject.id : undefined},
+        });
```

Comment:
@matejminar In Discover, for the `release` column, we're adding a per cell action that'll redirect to the release health page for a specific release. Can you confirm if this is right?

Reply:
Looks good 👍 Do we also need to preserve environment/date in the global header?

Reply:
I'm preserving the global selection header here now https://github.com/getsentry/sentry/pull/19230/commits/8377829f0904b26c295c472d744fbf0e84ba0b78